### ISSUE TITLE
Make Topics dropdown accessible for screenreaders

### ISSUE THREAD

**Issue by lizzycederberg** [usnationalarchives/nara-bia-photos#254]
On the [Topics page](https://www.archives.gov/research/native-americans/bia/photos/topics) (and potentially others) the user is unable to tab to the select dropdown. Upon further investigation it just seems that the user can tab to it, but there is no visual indication or screen reader indication they are there, so using the arrow keys to navigate the collapsed listbox is not intuitive. Users should be able to select items from the dropdowns and maintain focus while on the page.

This is in reference to the main "Select a Topic" dropdown in the header
![Screen Shot 2020-12-02 at 12 21 01 PM](https://user-images.githubusercontent.com/13699106/100907888-dbaaa880-3498-11eb-8813-1f67a1b35a3f.png)
